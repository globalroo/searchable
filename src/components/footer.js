import React from "react";
import { Link } from "@reach/router";
export const Footer = () => (
	<footer>
		<ul>
			<li>
				<Link to="movie/226486">Deep link Movie</Link>
			</li>
			<li>
				<Link to="person/60073">Deep link Person</Link>
			</li>
			<li>
				<Link to="tv/1399">Deep link TV</Link>
			</li>
			<li>TODO: Image Loader / 404 handling, Person and TV routes, icons in search menu and this page</li>
			<li>TODO: Add Grid list search</li>
		</ul>
	</footer>
);
