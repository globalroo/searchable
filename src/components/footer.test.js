import React from "react";
import { render, cleanup } from "react-testing-library";
import { Footer } from "./footer";
import "jest-dom/extend-expect";

describe("Asset Tile tests", () => {
	afterEach(cleanup);
	it("Renders the example Footer with some test links", () => {
		const { container } = render(<Footer />);
		expect(container).toMatchSnapshot();
	});
});
