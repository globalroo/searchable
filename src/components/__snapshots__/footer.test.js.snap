// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Asset Tile tests Renders the example Footer with some test links 1`] = `
<div>
  <footer
    class="small-text"
  >
    <h5>
      Examples:
    </h5>
    <hr />
    <a
      href="/movie/299536"
    >
      Deep link Movie
    </a>
     |
     
    <a
      href="/person/60073"
    >
      Deep link Alive Actor
    </a>
     |
     
    <a
      href="/person/194"
    >
      Deep link Deceased Actor
    </a>
     |
     
    <a
      href="/tv/1399"
    >
      Deep link TV
    </a>
     |
     
    <a
      href="/genres"
    >
      Most popular movies by Genre
    </a>
  </footer>
</div>
`;
